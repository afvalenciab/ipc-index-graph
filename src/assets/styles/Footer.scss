.footer {
  padding: 0px 50px;
  display: flex;
  justify-content: space-between;
  align-items: center;
  font-size: 12px;
  background: #F5F6F9;

  p {
    margin: 0;
    color: rgba(0,0,0,.8);
  }
}
