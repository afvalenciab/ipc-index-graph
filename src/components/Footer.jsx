import React from 'react';
import '../assets/styles/Footer.scss';

const Footer = () => {
  return (
    <footer className='footer'>
      <p>IPC Index - Indice de Precios y Cotizaciones.</p>
      <p>Hecho por Andres Valencia</p>
    </footer>
  );
};

export default Footer;
