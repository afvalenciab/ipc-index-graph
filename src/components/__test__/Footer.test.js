import React from 'react';
import { create } from 'react-test-renderer';
import Footer from '../Footer';
import Adapter from 'enzyme-adapter-react-16';
import { render, shallow, configure } from 'enzyme';

configure({ adapter: new Adapter() });

describe('Footer testing', () => {
  test('Match Snapshot', () => {
    const footer = create(<Footer />);
    expect(footer.toJSON()).toMatchSnapshot();
  });

  test('footer has 2 paragrah tags', () => {
    const footer = render(<Footer />);
    expect(footer.find('p')).toHaveLength(2);
  });

  test('Footer has class .footer', () => {
    const footer = shallow(<Footer />);
    const footerElm = footer.find('footer');
    expect(footerElm.hasClass('footer')).toBe(true);
  });
});