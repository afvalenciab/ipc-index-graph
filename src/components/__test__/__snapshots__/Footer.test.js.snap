// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer testing Match Snapshot 1`] = `
<footer
  className="footer"
>
  <p>
    IPC Index - Indice de Precios y Cotizaciones.
  </p>
  <p>
    Hecho por Andres Valencia
  </p>
</footer>
`;
